ui_framework/components/tabs/tabs.js

```diff
@@ -0,0 +1,116 @@
+import React, { cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTab } from './tab';
+
+export class KuiTabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedTabIx: this.getStartSelectedIx()
+    };
```

Comment:
Also, we generally try to avoid using abbreviations like this, so we should write out `index` instead of `ix`.